sys-apps/ethq/ethq-0.5.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV=$(ver_rs 1-2 '_')
```

Comment:
Doesn't that replace all separators? Why not just `${PV//./_}` then?

Reply:
Replaced.